### ISSUE TITLE
bot: postgresql output bot - problem on python3

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq#453]
Line 53 in postgresql/output.py:

```
        values = event.values()
```

Works in python2 but not in python3. Reason: the values() function returns a view on a dict in python3. But in python 2 a list.
Solution:

```
        values = list(event.values())
```

**Comment by sebix** [comment]
Where does the bot actually crash? In `self.cur.execute(query, values)` or before? If psycopg2 throws the error, it could bring that topic to the psycopg2 project about supporting any iterables instead of only lists.

**Comment by aaronkaplan** [comment]
execute

**Comment by mauroasilva** [comment]
Yeah indeed... If the list constructor actually accepts it, it is probably
an iterable. I think it would be good to give them the heads up.

**Comment by dmth** [comment]
I ran into that problem yesterday. Also fixed it with converting `values` to a list:

```
--- a/intelmq/bots/outputs/postgresql/output.py
+++ b/intelmq/bots/outputs/postgresql/output.py
@@ -58,7 +58,7 @@ class PostgreSQLBot(Bot):
         self.logger.debug('Query: {!r} with values {!r}'.format(query, values))
         try:
             # note: this assumes, the DB was created with UTF-8 support!
-            self.cur.execute(query, values)
+            self.cur.execute(query, list(values))
         except (psycopg2.InterfaceError, psycopg2.InternalError,
                 psycopg2.OperationalError, AttributeError):
             try:
```

As our deployment is targeted on Python3 we would create a PR for this.
Do you see any side-effects?

**Comment by sebix** [comment]
A fix is included in PR #465, which I just merged as there came no objections or comments.

**Comment by aaronkaplan** [comment]
thx
